pstatus-notifications-workflow-ktor/build.gradle.kts

```diff
@@ -62,6 +65,8 @@ dependencies {
     implementation ("io.ktor:ktor-client-content-negotiation:2.1.0")
     implementation ("io.netty:netty-all:4.1.68.Final")
     implementation ("io.netty:netty-tcnative-boringssl-static:2.0.52.Final:windows-x86_64")
+    implementation("software.amazon.awssdk:dynamodb-enhanced:2.20.70") // Latest Enhanced Client for DynamoDB
```

Comment:
Should be able to remove these two lines as well for dynamodb.

Reply:
This is required for the dynamo db annotation for the response class. The other one has been removed.